.github/workflows/NUnit.CI.yml

```diff
@@ -30,6 +30,7 @@ jobs:
         dotnet-version: |
             6.0.x
             8.0.x
+            10.0.x
```

Comment:
Please add 9.0.x, It was previously found using the global.json file.
Suggest to drop 10.0.x That way if we move from rc to production we don't need to update this file.